Harvest.Web/Controllers/Api/TicketController.cs

```diff
@@ -135,6 +140,11 @@ public async Task<ActionResult> Get(int projectId, int ticketId)
                         Attachments = a.Attachments.Select(b => new TicketAttachment{Id = b.Id, CreatedBy = b.CreatedBy, CreatedOn = b.CreatedOn, FileName = b.FileName, Identifier = b.Identifier}).ToList()

```

Comment:
Instead of doing the foreach below, I think you can do it in here:
....... , SasLink = fileService.GetDownloadUrl(storageSettings.ContainerName, b.Identifier).AbsoluteUri }).ToList()